hw03_diopa.py

```diff
@@ -0,0 +1,70 @@
+######################################################################
+# Author: Abdou Diop             TODO: Change this to your name, if modifying
+# Username: diopa                      TODO: Change this to your username, if modifying
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Draw a colorful flower using turtle and random colors
+######################################################################
+# Acknowledgements:
+
+# licensed under a Creative Commons
+# Attribution-Noncommercial-Share Alike 3.0 United States License.
+#################################################################################
+
+import turtle
+import random as rand
+
+def main():
```

Comment:
This code is missing two requirements from the assignment prompt: Proper use of functions and contains a loop.

Reply:
I.e., a single function `main` is not enough, and a loop that only iterates once doesn't demonstrate an understanding of loops.